src/cljs/rems/administration/create_form.cljs

```diff
@@ -619,28 +619,29 @@
                 [:li (text-format :t.create-form/column-n (inc column-id))
                  [:ul
                   (when (:key column-errors)
-                    (format-validation-link (str "fields-" field-index "-column-" column-id "-key")
+                    (format-validation-link (str "fields-" field-index "-columns-" column-id "-key")
                                             (text-format (:key column-errors) (text :t.create-form/option-key))))
                   (into [:<>]
                         (for [[lang error] (:label column-errors)]
-                          (format-validation-link (str "fields-" field-index "-column-" column-id "-label-" (name lang))
+                          (format-validation-link (str "fields-" field-index "-columns-" column-id "-label-" (name lang))
                                                   (text-format error (str (text :t.create-form/option-label)
                                                                           " (" (.toUpperCase (name lang)) ")")))))]]))))]))
 
 (defn format-validation-errors [form-errors form lang]
   ;; TODO: deduplicate with field definitions
   (into [:ul
-         (when (:organization form-errors)
-           [:li [:a {:href "#" :on-click (focus-input-field "organization")}
-                 (text-format (:organization form-errors) (text :t.administration/organization))]])
-
-         (when (:form/internal-name form-errors)
-           [:li [:a {:href "#" :on-click (focus-input-field "internal-name")}
-                 (text-format (:form/internal-name form-errors) (text :t.administration/internal-name))]])
-
-         (when (:form/external-title form-errors)
-           [:li [:a {:href "#" :on-click (focus-input-field "external-title")}
-                 (text-format (:form/external-title form-errors) (text :t.administration/external-title))]])]
+         (when-let [error (:organization form-errors)]
+           (format-validation-link "organization-dropdown"
+                                   (text-format error (text :t.administration/organization))))
+
+         (when-let [error (:form/internal-name form-errors)]
+           (format-validation-link "internal-name"
+                                   (text-format error (text :t.administration/internal-name))))
+
+         (for [[lang error] (:form/external-title form-errors)]
+           (format-validation-link (str "external-title-" (name lang))
+                                   (text-format error (str (text :t.administration/external-title)
+                                                           " (" (.toUpperCase (name lang)) ")"))))]
```

Comment:
I think we did discuss this one with @Macroz at some point and he suggested to not modify text in code but rather do it in localisations

Reply:
I'm repeating the style used in `format-field-validation` above. I agree that it's not that pretty, but at least it's consistent within this ns...

Reply:
Actually, I have an idea for a small refactor to DRY this out a bit

Reply:
sure, I just pointed that out since it is still fresh in my memory, I am ok with the code otherwise

Reply:
Yes, ideally the parentheses would be in the localization text, however we have not done here so before for these cases.

Reply:
no need to change anything from my side

Reply:
This whole thing is a bit of a hack, and the root cause seems to be #1334 . The form editor uses this style of localized fields:

```
Title
FI    ____________
EN  ____________
SV  ____________
```

This isn't a problem when showing validation errors next to the fields, but when generating this summary for the top of the page we need to somehow textually identify the subfields for various languages.

This wouldn't be a problem if we used this more verbose style that some other administration pages use:

```
Title (FI) _________
Title (EN) ________
Title (SV) ________
```

Reply:
Anyway, I pushed a refactor that at least isolates this string-building logic to one place.

Reply:
I see your point, I agree